### PR TITLE
[TACHYON-1636] Add the Chinese version of documentation page 'Configuring-Tachyon-with-Swift'

### PR DESCRIPTION
**Pull request by saltylin**
https://tachyon.atlassian.net/browse/TACHYON-1636
In `Configuring-Tachyon-with-Swift.md`, it writes "it is adviced to set value of `<swift-auth-model>`  to `false`". However, possible values of `<swift-auth-model>` are `keystone`, `tempauth`, `swiftauth`. Is this a typo and what it wants to say is setting value of `<swift-use-public>` to `false`?

### CODE REVIEW

File: docs/cn/Configuring-Tachyon-with-Swift.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+{% include Configuring-Tachyon-with-Swift/stop-tachyon.md %}
+
+# 对IBM SoftLayer进行功能测试
+
+在`tests/pom.xml`配置你的Swift或者SoftLayer账户，其中`authMethodKey`的值应为`keystone`、`tempauth`或`swiftauth`，要进行功能测试，运行：
+
+{% include Configuring-Tachyon-with-Swift/functional-tests.md %}
+
+若测试失败，日志记录在`tests/target/logs`下。可以通过以下命令激活堆转储：
```
Comment (ifcharming) [inline docs/cn/Configuring-Tachyon-with-Swift.md]: heap dump -> "抓取堆状态备份“ ?

Not sure which translation is better for this term, it's your call :)

**Comment by ifcharming** [comment]
LGTM

**Comment by saltylin** [comment]
Could anyone verify this?
In `Configuring-Tachyon-with-Swift.md`, it writes "it is adviced to set value of `<swift-auth-model>`  to `false`". However, possible values of `<swift-auth-model>` are `keystone`, `tempauth`, `swiftauth`. Is this a typo and what it wants to say is setting value of `<swift-use-public>` to `false`?

**Comment by jsimsa** [comment]
yes, it should be `<swift-use-public>` not `<swift-auth-model>`

**Comment by yupeng9** [comment]
@saltylin , could you also fix the typo Jiri mentioned in this doc? Thanks.

**Comment by saltylin** [comment]
Fixed the typo.

**Comment by yupeng9** [comment]
Thanks!

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**